controllers/release/release_adapter_test.go

```diff
@@ -70,7 +70,14 @@ var _ = Describe("Release Adapter", Ordered, func() {
 			},
 			Spec: ecapiv1alpha1.EnterpriseContractPolicySpec{
 				Description: "test-policy-description",
-				Sources:     []string{"foo"},
+				Sources: []ecapiv1alpha1.PolicySource{
+					ecapiv1alpha1.PolicySource{
```

Comment:
Source not PolicySource

Reply:
Inside of the struct, you don't want a GitRepository like this, you want (at most)
```
Name: string
Policy: 
- string
Data: 
- string
```

Reply:
Fixed